index.rst

```diff
@@ -105,7 +105,7 @@ Force balance tests
 
   *Results*: After applying different forces at 10% to 100% performance, maximum are occasionally seen at above 1000N, whereas the limit currently stands at 900 N.
 
-  *Pending tasks*: Update technote to reflect most recent results (shown at the Glass Safety review) and merge into main. Define next steps, and possible risk minimization. Technote format is old.
+  *Pending tasks*: Update technote to reflect most recent results (shown at the Glass Safety review) and merge into main. Define next steps, and possible risk minimization. Technote format is old. There is a ticket (`SITCOM-1463 <https://sitcomtn-092.lsst.io/v/SITCOM-1463/index.html>`) to change the format but although it shows as done, the ticket information does not appear.
```

Comment:
Can you check with the authors why is this the case? (why technote 92 does not appear updated to the new format?)

Reply:
Marina Pavlovic on ticket SITCOM-1463 migrated the tecnote to the new format but she did not copy the information. The current version shown still has the old format but I don't know why.